packages/testing/src/render.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react'
+import { render } from '@testing-library/react'
+import { MockedProvider } from '@apollo/react-testing'
+
+import Routes from '~/web/Routes'
```

Comment:
I think we might be able to rely on a dynamic import here, and using `getPaths().web.routes`. If not, maybe we could configure the configuration to use `getPaths` because I want to have a single handle on the location of this file.